# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/moz.build
# Commit: e65d7e9dea10
# Full Hash: e65d7e9dea10c417588224abb452aa2d181e8e0c
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-01-13 15:45:24
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Convert IDBCursor and BackgroundCursorChild to templates to increase type safety and reduce state. r=dom-workers-and-storage-reviewers,ytausky
#   
#   This also simplifies delegating calls that are dependent on the cursor type.
#   
#   Also reduce dependency on IDBCursor.h by moving enums and type traits to IDBCursorType.h
# ==============================================================================

diff -r 422de232c5ce -r e65d7e9dea10 dom/indexedDB/moz.build
--- a/dom/indexedDB/moz.build	Fri Jan 10 15:59:51 2020 +0000
+++ b/dom/indexedDB/moz.build	Fri Jan 10 15:23:52 2020 +0000
@@ -26,6 +26,7 @@
 EXPORTS.mozilla.dom += [
     'FlippedOnce.h',
     'IDBCursor.h',
+    'IDBCursorType.h',
     'IDBDatabase.h',
     'IDBEvents.h',
     'IDBFactory.h',
@@ -39,6 +40,7 @@
     'IDBTransaction.h',
     'IndexedDatabase.h',
     'IndexedDatabaseManager.h',
+    'InitializedOnce.h',
 ]
 
 EXPORTS.mozilla.dom.indexedDB += [
@@ -56,6 +58,7 @@
     'FileInfo.cpp',
     'FileSnapshot.cpp',
     'IDBCursor.cpp',
+    'IDBCursorType.cpp',
     'IDBDatabase.cpp',
     'IDBEvents.cpp',
     'IDBFactory.cpp',
